backend/core/src/listings/listings.service.ts

```diff
@@ -43,7 +43,7 @@ export class ListingsService {
       .leftJoin("property.units", "units")
       .leftJoin("units.unitType", "unitTypeRef")
       .groupBy("listings.id")
-      .orderBy({ "listings.id": "DESC" })
+      .orderBy({ "listings.createdAt": "DESC" })
```

Comment:
Does this change anything useful, since this is for the inner listings.id IN (...) query on line 76, I don't think it actually changes the sort order, which is handled above that. Did you confirm that?